testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -4777,6 +4777,23 @@ QUnit.module('keyboard navigation', {
         assert.ok($cancelButton.hasClass('dx-state-focused'), 'cancel button is focused');
     });
 
+    QUnit.testInActiveWindow('Popup has open state if tab key was pressed when applyValueMode: "instantly"', function(assert) {
+        if(devices.real().deviceType !== 'desktop') {
+            assert.ok(true, 'desktop specific test');
+            return;
+        }
+
+        this.dateBox.option({
+            type: 'datetime',
+            opened: true,
+            applyValueMode: 'instantly',
+        });
+
+        this.keyboard.keyDown('tab');
+
+        assert.ok(this.dateBox.option('opened'), 'Popup open');
```

Comment:
"Popup open' doesn't seem correct to me. 'Popup still opened' or smth like this. Also u can just do not add this text at all - no text - no mistakes :) Test name is enough to understand the test

Reply:
Fixed in [99a178d2a43ec5be8ad8dd319a440caaa9118077].